winui/ComboBox/Highlighting.md

```diff
@@ -0,0 +1,204 @@
+                            TextHighlightMode="Matched"                                
+                            TextSearchMode="StartsWith"                               
+                            HighlightedTextFontSize="15"                               
+                            HighlightedTextFontStyle="Italic"                               
+                            HighlightedTextFontWeight="Medium"                                
+                            HighlightedTextForeground="Red"                                
+                            ItemsSource="{Binding SocialMedias}">       
+    </editors:SfComboBox>
+
+{% endhighlight %}
+{% highlight C# %}
+        comboBox.DisplayMemberPath = "Name";
+        comboBox.IsEnabled=true;
+        comboBox.IsFilteringEnabled = true;          
+        comboBox.TextHighlightMode = ComboBoxTextHighlightMode.Unmatched;
+        comboBox.TextSearchMode = ComboBoxTextSearchMode.Contains;
+        comboBox.HighlightedTextFontSize = 10;
+        comboBox.HighlightedTextFontStyle = FontStyle.Normal;
+        comboBox.HighlightedTextFontWeight = FontWeights.ExtraLight;
+        comboBox.HighlightedTextForeground = Color.Red;
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI ComboBox text highlighting properties](Highlighting_images/winui-combobox-highlightingproperties.png)
+
+
+
+### **Highlight beginning text**
+It highlights the matches that starts with the typed characters in suggestion list.
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfComboBox     x:Name="comboBox"                        
+                        DisplayMemberPath="Name" 
+                        IsEditable="true"
+                        IsFilteringEnabled="True"                            
+                        TextHighlightMode="Matched"
+                        TextSearchMode="StartsWith"                           
+                        HighlightedTextForeground="Red"                               
+                        ItemsSource="{Binding Countries}">
+</editors:SfComboBox>
+{% endhighlight %}
+{% highlight C# %}
+
+comboBox.DisplayMemberPath = "Name";
+comboBox.IsEnabled=true;
+comboBox.IsFilteringEnabled = true;
+comboBox.TextHighlightMode = ComboBoxTextHighlightMode.Matched;
+comboBox.TextSearchMode = ComboBoxTextSearchMode.StartsWith;
+comboBox.HighlightedTextForeground = Color.Red;
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Searching with one character
+
+![WinUI ComboBox text highlighting based on begining text one character](Highlighting_images/winui-combobox-textsearchmode-startswith-onecharacter.png)
+
+### Searching with more than one character
+
+![WinUI ComboBox text highlighting based on begining text more than one character](Highlighting_images/winui-combobox-textsearchmode-startswith-morethanonecharacter.png)
+
+
+
+
+### **Highlight all occurance of search text**
+It highlights all the matches that contains the typed characters in suggestion list. 
```

Comment:
It highlights all the matches that contain the typed characters in the suggestion list.